api/workflow.proto

```diff
@@ -131,6 +131,12 @@ service WorkflowService {
             put: "/apis/v1beta1/{namespace}/workflow_templates/{uid}/archive"
         };
     }
+
+    rpc GetArtifact (GetArtifactRequest) returns (ArtifactResponse) {
+        option (google.api.http) = {
+            get: "/apis/v1beta1/{namespace}/workflow_executions/{name}/artifact"
```

Comment:
should this endpoint include the artifact names in `URL`? also `artifact` => `artifacts`?